src/FSharp.Stats/Testing/UTest.fs

```diff
@@ -0,0 +1,51 @@
+﻿namespace FSharp.Stats.Testing
+
+// taken/implemented from: https://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U_test#U_statistic
+module UTest =
+
+    open FSharp.Stats
+    open FSharp.Stats.Testing
+
+    // TO DO:   Bergmann et al. (2000) showed that there are different implementations of this test that lead to different results.
```

Comment:
Please check if your computations are correct before PR can be merged. Additionally, check your results against established packages in R, SPSS or puplications rather than volatile wikipedia chapters.